src/main/kotlin/no/nav/familie/ef/iverksett/patch/PatchStartdatoService.kt

```diff
@@ -0,0 +1,103 @@
+package no.nav.familie.ef.iverksett.patch
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import no.nav.familie.ef.iverksett.iverksetting.domene.IverksettType
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.security.token.support.core.api.Unprotected
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+
+@RestController
+@Unprotected
+@RequestMapping("api/patch-startdato")
+class PatchStartdatoController(private val patchStartdatoService: PatchStartdatoService) {
+
+    @GetMapping
+    fun patch(@RequestParam oppdaterVedtak: Boolean = false) {
+        patchStartdatoService.patch(oppdaterVedtak)
+    }
+}
+
+@Service
+class PatchStartdatoService(private val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) {
+
+    private val logger: Logger = LoggerFactory.getLogger(javaClass)
+
+    @Transactional
+    fun patch(oppdaterVedtak: Boolean) {
+        val data = this::class.java.classLoader.getResourceAsStream("patch-startdato.csv")!!.bufferedReader().readLines()
+                .map { line -> line.split(",").let { UUID.fromString(it[0]) to LocalDate.parse(it[1]) } }
```

Comment:
Trenger vi å parse datoen for så å skrive den tilbake til ISO_LOCAL_DATE ? Eller kan den egentlig bare behandles som en streng hele veien? Ref linje 73

Reply:
God poeng, sender med som string, og verifiserer at det er "gyldig dato"